test/Microsoft.AspNet.Mvc.Core.Test/FileResultTest.cs

```diff
@@ -120,6 +121,42 @@ public void Constructor_SetsContentType()
             Assert.Equal("attachment; filename=filename.ext; filename*=UTF-8''filename.ext", httpContext.Response.Headers["Content-Disposition"]);
         }
 
+        [Fact]
+        public void ExecuteResultAsync_ThrowsException_IfCannotResolveLoggerFactory()
+        {
+            // Arrange
+            var httpContext = new DefaultHttpContext();
+            httpContext.RequestServices = new ServiceCollection().BuildServiceProvider();
+            var actionContext = CreateActionContext(httpContext);
+            var result = new EmptyFileResult("application/my-type");
+
+            // Act
+            Action act = () => result.ExecuteResultAsync(actionContext).Wait();
```

Comment:
Can do this inline in the `Assert.Throws<...>()`. Kind of weird to define an Action for this. Then just change to comment to `// Act & Assert`.